cosmos/operators/local.py

```
@@ -69,12 +69,12 @@ def __init__(
         self.should_store_compiled_sql = should_store_compiled_sql
         super().__init__(**kwargs)
 
-    @cached_property
-    def subprocess_hook(self):
+    @cached_property  # type: ignore[misc] # ignores internal untyped decorator
```

Is there an alternative to ignore this error?